packages/account/src/Sections/Profile/PhoneVerification/cancel-phone-verification-modal.tsx

```diff
@@ -16,6 +17,16 @@ const CancelPhoneVerificationModal = observer(() => {
     const { setVerificationCode, is_virtual } = client;
     const { isMobile } = useDevice();
 
+    useEffect(() => {
+        if (show_modal) {
+            //@ts-expect-error will remove this error when Analytics package types are being updated
+            Analytics.trackEvent('ce_phone_verification_form', {
```

Comment:
as I see we call trackevents method with the same params in many places, would it be better to create one helper function and just pass via params the different values?